code/game/objects/items/devices/laserpointer.dm

```diff
@@ -135,7 +135,8 @@
 
 	//catpeople
 	for(var/mob/living/carbon/human/H in view(1,targloc))
-		if(!isfelinid(H) || H.incapacitated() || H.is_blind())
+		//if(!isfelinid(H) || H.incapacitated() || H.is_blind())
+		if(!(isfelinid(H) || HAS_TRAIT(H, TRAIT_FELINE)) || H.incapacitated() || H.is_blind()) //SKYRAT EDIT - FELINE TRAIT
```

Comment:
```suggestion
		if(!isfelinid(H) || !HAS_TRAIT(H, TRAIT_FELINE) || H.incapacitated() || H.is_blind()) //SKYRAT EDIT - FELINE TRAIT
```

I feel like the way you wrote it is kinda unclear. Maybe just me though.

Reply:
This is clearer at a glance, don't see no reason why not to use it :D
You even save a character.

Reply:
without the parenthesizes, it means that you have to be a felinid AND have the trait to be affected, meaning it completely breaks it